flink-core/src/main/java/org/apache/flink/api/java/typeutils/TypeExtractor.java

```diff
@@ -1723,20 +1720,17 @@ private boolean isValidPojoField(Field f, Class<?> clazz, ArrayList<Type> typeHi
 					// return type is void.
 					m.getReturnType().equals(Void.TYPE)
 				) {
-					if(hasSetter) {
-						throw new IllegalStateException("Detected more than one setter");
-					}
 					hasSetter = true;
 				}
 			}
 			if(hasGetter && hasSetter) {
 				return true;
 			} else {
 				if(!hasGetter) {
-					LOG.debug(clazz+" does not contain a getter for field "+f.getName() );
+					LOG.info(clazz+" does not contain a getter for field "+f.getName() );
```

Comment:
I would say yes. It is very hard for users to figure out why a object is not a POJO. Information about the problematic field is helpful. This message will occur as often as `clazz is not a valid POJO.`, so one output per non-POJO.